### ISSUE TITLE
OpenSC build error in opensc-notify

### ISSUE THREAD

**Issue by misterzed88** [OpenSC/OpenSC#2068]
### Problem Description

OpenSC fails to build from the master branch, with error:
```
  CCLD     opensc-notify
/usr/bin/ld: opensc_notify-opensc-notify.o: undefined reference to symbol 'pthread_create@@GLIBC_2.2.5'
//lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

The problem is that opensc_notify uses pthread functions without linking with the library.

The build was done with:
`gcc version 6.3.0 20170516 (Debian 6.3.0-18+deb9u1)`

### Proposed Resolution
```
--- a/src/tools/Makefile.am
+++ b/src/tools/Makefile.am
@@ -98,7 +98,7 @@ endif
 
 opensc_notify_SOURCES = opensc-notify.c opensc-notify-cmdline.c
 opensc_notify_LDADD = $(top_builddir)/src/libopensc/libopensc.la $(OPTIONAL_NOTIFY_LIBS)
-opensc_notify_CFLAGS = -I$(top_srcdir)/src $(OPTIONAL_NOTIFY_CFLAGS)
+opensc_notify_CFLAGS = -I$(top_srcdir)/src $(PTHREAD_CFLAGS) $(OPTIONAL_NOTIFY_CFLAGS)
 opensc_notify_CFLAGS += -Wno-unused-but-set-variable
 if HAVE_UNKNOWN_WARNING_OPTION
 opensc_notify_CFLAGS += -Wno-unknown-warning-option
```

**Comment by Jakuje** [comment]
The proposed change looks good. Can you submit a PR? Wondering why it does not fail in CI? Is the notify support built in CI at all?

**Comment by misterzed88** [comment]
PR submitted.

I have no why the error doesn't show up in a CI build. My manual build was done using plain bootstrap and configure (without any options). Is there anything in the env. setup of the CI that makes a difference, making it include the pthread lib by default?

**Comment by Jakuje** [comment]
Fixed with #2069.
